pdffigures.cpp

```diff
@@ -29,6 +29,8 @@ void printUsage() {
          "prefix-<page#>.png\n");
   printf("-o, --save-figures <prefix>: Save images of detected figures to "
          "prefix. Files are save to prefix-<(Table|Figure)>-<Number>.png\n");
+  printf("-c, --save-color-images: Additionally save color images for figures and tables."
```

Comment:
I would prefer that "-c" takes its own unique prefix like "-j" and "-o" does. Otherwise "-c" can only be used in conjunction with "-o", and I think in practice most users would not want to have both the color and non-color versions of the images.